src/main/java/uk/ac/ebi/ena/webin/cli/WebinCli.java

```diff
@@ -112,8 +112,6 @@ public class WebinCli {
             switch (ex.getErrorType()) {
                 case USER_ERROR:
                     return USER_ERROR;
-                case VALIDATION_ERROR:
-                    return VALIDATION_ERROR;
```

Comment:
This will now throw a system error against a validation error. will this not affect unit and/or integration tests?

Reply:
VALIDATION_ERROR has been removed from WebinCliException.

Reply:
Restored VALIDATION_ERROR for user errors from Webin-CLI-validator.